spring-cloud-gcp-core/src/main/java/org/springframework/cloud/gcp/core/UsageTrackingHeaderProvider.java

```diff
@@ -49,7 +78,7 @@ public UsageTrackingHeaderProvider(Class clazz) {
 
 		headers.put("User-Agent",
 				"Spring/" + this.getClass().getPackage().getImplementationVersion()
```

Comment:
What about this line?

Reply:
Is this one supposed to always identical to the other? 

To confirm, there are two components  "Spring/" and "spring-cloud-gcp/" 
The former uses "this.getClass()" while the latter uses "this.clazz". The second is the one I replaced. Is the first one also supposed to be replaced identically, or is it supposed to potentially show the different repackaged version?

Reply:
I think the idea was that some modules within the project may have different versions. This would actually mean that you need to generated this file separately for each module.
@joaoandremartins